pyproject.toml

```diff
@@ -7,7 +7,7 @@ name = "aleph-vm"
 dynamic = ["version"]
 description = "Aleph.im VM execution engine"
 readme = "README.md"
-requires-python = ">=3.9"
+requires-python = "==3.9"
```

Comment:
I think that this cannot be added, because new Debian versions comes with Python 3.11 by default.

Reply:
Indeed, I went a bit too hasty.